src/pretix/base/settings.py

```diff
@@ -1914,6 +1914,8 @@ def unserialize(cls, s):
 
 your order {code} for {event} has been canceled.
 
+{comment}
```

Comment:
I think we might need a manual adjustment of the .po-files to include this in all languages, so weblate does not flag all these strings as new/untranslated